introduction.md

```diff
@@ -5,3 +5,12 @@ This is the part where I introduce myself.
 
 Ruby is my favorite Code School path.
 
+### Skills
+
+* I can play Fruit Ninja in real life
+* I can make sarcastic jokes (sarcasm)
+* I can do all of the above while making a handstand
+* I can make a handstand while making a handstand
+
+Peter Piper picked a peck of pickled peppers and f***ed your b****.
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?